.circleci/config.yml

```diff
@@ -301,4 +301,4 @@ workflows:
       - py39_linux
       - unittests_gpu17_cuda10
       - unittests_gpu17_cuda11
-      - test_cuda11_wheel
+      # - test_cuda11_wheel
```

Comment:
Not sure why enabling this makes CI fail? We should reenable before merging or is the plan fix this part in a separate PR?